airflow/providers/apache/kafka/hooks/base.py

```diff
@@ -0,0 +1,71 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from typing import Any
+
+from confluent_kafka.admin import AdminClient
+
+from airflow.hooks.base import BaseHook
+
+
+class KafkaHook(BaseHook):
+    """A base hook for interacting with Apache Kafka"""
+
+    conn_name_attr = "kafka_config_id"
+    default_conn_name = "kafka_config_default"
+    conn_type = "kafka"
+    hook_name = "Apache Kafka"
+
+    def __init__(self, kafka_config_id=default_conn_name, *args, **kwargs):
+        """Initialize our Base"""
+        super().__init__()
+        self.kafka_config_id = kafka_config_id
+        self.get_conn()
```

Comment:
Do we need to call this here, is it here for validation?

Reply:
It's purely for validation.